.gitpod.yml

```diff
@@ -35,10 +29,13 @@ ports:
 # some useful extensions to have
 vscode:
   extensions:
-    - eamodio.gitlens@10.2.3:CRhSs/nkGIS3K5yCBlhVTg==
-    - njpwerner.autodocstring@0.5.4:TKRAH5+nBrBnq+ysTYA86Q==
-    - lextudio.restructuredtext@141.0.0:JobzqAU1ylF4bDdKqnrpOg==
-    - ritwickdey.liveserver@5.6.1:5Mhl1RgVVDcIym163x8r+w==
+    - eamodio.gitlens
+    - njpwerner.autodocstring
+    - lextudio.restructuredtext
+    - ritwickdey.liveserver
+    - ms-python.python
+    - yzhang.markdown-all-in-one
+    - bungcip.better-toml
 
```

Comment:
Unfortunately there doesn't seem to be any extension with Cython syntax highlighting support. https://github.com/guyskk/language-cython-for-vscode is the only one, but you can't install it in Gitpod.

Reply:
The last commit in the extension is over 2 years old. It might see if I can make a port to OSVX which is the marketplace used by Gitpod

Reply:
Not sure it's worth that much effort. I mean specifically for the sprint or SciPy - good Cython highlighting in VSCode seems like an obvious gap that could/should be improved.